include/rootfs.mk

```diff
@@ -115,6 +121,8 @@ define prepare_rootfs
 	rm -rf \
 		$(1)/boot \
 		$(1)/tmp/* \
+		$(1)/lib/apk/db/scripts.tar \
```

Comment:
We will still need `scripts.tar` for all the other scripts, to be run on removal or upgrade of the package.

Reply:
I will just edit the tar and remove all the relevant script but I opened the scripts.tar and there were only post-install and pre-deinstall.

Ok i guess pre-deinstall should not be removed...